SOURCES/forkexecd.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Manages subprocesses independently from a multithreaded program.
+After=syslog.target
+Wants=syslog.target
+
+[Service]
+Type=notify
```

Comment:
Yes, we need this one to be of type notify, because it does not use the message-switch. We could change it to simple + socket activation later (or perhaps throw it away altogether...).